docs/configuration.md

```diff
@@ -1,4 +1,28 @@
+# Configuration
+
+## ARM mode
+
+ARM mode requires setting up service principal for authentication and permissioning. For setting up a service principal from the command line please refer
```

Comment:
Please mention about `azure login` command too.  If the user runs that command, then there's no need to set up service principal.

Reply:
Interesting, I never knew `azure login` can get ARM mode credentials working... We're trying to move everything Azure CLI 2.0, do you know if this works with `az login` ?

Reply:
We had added support for `azure login` only. I doubt if `az login` will work here. However if the credential management of both `azure login` and `az login` is same, then it may work. But we'll have to verify.
For now we are sure about the support for `azure login`.

Reply:
@binamov , waiting for this change to merge your PR :)